pnpm-lock.yaml

```diff
@@ -16239,16 +16218,16 @@ snapshots:
       - '@emotion/react'
       - '@types/react'
 
-  '@mantine/core@7.17.4(@mantine/hooks@7.17.4(react@19.0.0))(@types/react@19.0.2)(react-dom@19.0.0(react@19.0.0))(react@19.0.0)':
+  '@mantine/core@8.0.0-alpha.3(@types/react@19.0.2)(react-dom@19.0.0(react@19.0.0))(react@19.0.0)':
     dependencies:
       '@floating-ui/react': 0.26.28(react-dom@19.0.0(react@19.0.0))(react@19.0.0)
-      '@mantine/hooks': 7.17.4(react@19.0.0)
+      '@mantine/hooks': 8.0.0-alpha.3(react@19.0.0)
```

Comment:
this was the issue with aliasing. it was using v6 mantine hooks since they are in the same package now